lib/active_stash.rb

```diff
@@ -1,4 +1,4 @@
-require "active_stash/version"
+require "git-version-bump"
```

Comment:
This will require `git-version-bump` to be a runtime dependency, rather than a development dependency.

Reply:
i'll update. 👍🏼